lib/api/core/models/security/role.js

```diff
@@ -12,6 +12,19 @@ function Role () {
   this.closures = {};
 }
 
+function isIndexExist(requestObject, context) {
```

Comment:
nitpicking: isIndexExisting (or doesIndexExist ; I am not sure we should really need to strictly comply to the isxxx convention for  all methods that return a boolean).